site/content/guides/routing.md

```diff
@@ -0,0 +1,285 @@
+          <Route path="/about" component={AboutPage} />
+          <Route component={NotFoundPage} />
+        </Switch>
+      </div>
+    </div>
+  );
+};
+
+```  
+
+Now whenever you navigate to URL that doesn't match `/` or `/about` you'll see `NotFoundPage`
+rendered.
+
+Note: in `Switch` only the first matching `Route` gets rendered.
+
+## Nested routes
+
+It is also possible for child components to define their own subroutes. So for example if you want
+`AboutPage` to have subpages you can render `Link` and `Route` to it.
+
+```js
+...
+const AboutUs = () => <article>Some content about us...</article>;
+const AboutThem = () => <article>Some content about them...</article>;
+
+const AboutPage = ({ match }) => {
+  return (
+    <div>
+      <h2>About page</h2>
+      <ul>
+        <li><Link to={`${match.url}`}>About</Link></li>
+        <li><Link to={`${match.url}/us`}>About us</Link></li>
+        <li><Link to={`${match.url}/them`}>About them</Link></li>
+      </ul>
+      <Switch>
+        <Route path={`${match.url}/us`} component={AboutUs} />
+        <Route path={`${match.url}/them`} component={AboutThem} />
+      </Switch>
+    </div>
+  );
+};
+...
+```
+
+Because `AboutPage` is rendered using `Route` component it receives a `match` object as a prop. This
+object contains `url` which let's you define routes relative to the current.  
+
+It is important that the parent `Route` doesn't have `exact` prop so that it would match all the URLs
+starting with the `path`.
+
+## Route params
+
+In your path you can also define params which will be passed to your component once the path is 
+matched. With params previous example can be rewritten in the following way: 
+
+```js
+const AboutAnyone = ({ match }) => <article>Some content about {match.params.party}...</article>;
+
+const AboutPage = ({ match }) => {
+  return (
+    <div>
+      <h2>About page</h2>
+      <ul>
+        <li><Link to={`${match.url}`}>About</Link></li>
+        <li><Link to={`${match.url}/us`}>About us</Link></li>
+        <li><Link to={`${match.url}/them`}>About them</Link></li>
+      </ul>
+      <Switch>
+        <Route path={`${match.url}/:party`} component={AboutAnyone} />
+      </Switch>
+    </div>
+  );
+};
+```
+
+## Another example
+
+For more information about usage of Frint routing you can take a look at example here: 
+https://github.com/Travix-International/frint/tree/master/examples/router 
+
+To launch it clone `frint` repository
+
+```
+git clone git@github.com:Travix-International/frint.git
+```
+
+Go to `frint` directory, install npm dependencies and bootstrap internal dependencies
+
+```
+$ cd frint
+$ npm install
+$ npm run bootstrap
+```
+
+Go to the `examples/router` directory which has the example's source code, install npm dependencies
+and start the project
+
+```
+$ cd examples/router
+$ npm install
+$ npm start
+```
+
+It will open a new browser for you with running the example app.
```

Comment:
my English is quite limited, but may be it can be worded a bit better:

```
It will launch the example in your browser for you, after taking care of bundling.
```